openedx/features/badging/tests/test_views.py

```diff
@@ -1,50 +1,44 @@
 import mock
-
 from django.core.urlresolvers import reverse
-from django.http import Http404
-from django.test import RequestFactory
 from django.test.client import Client
-
+from lms.djangoapps.onboarding.tests.factories import UserFactory
 from openedx.core.djangoapps.xmodule_django.models import CourseKeyField
+from openedx.core.djangolib.testing.philu_utils import (clear_philu_theme,
+                                                        configure_philu_theme)
 from student.tests.factories import CourseEnrollmentFactory
-from student.tests.factories import UserFactory
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory
 
 from .factories import BadgeFactory, UserBadgeFactory
-from .. import views as badging_views
 
 
 class BadgeViewsTestCases(ModuleStoreTestCase):
 
     def setUp(self):
         super(BadgeViewsTestCases, self).setUp()
         self.course = CourseFactory(org="test", number="123", run="1")
-        self.request_factory = RequestFactory()
         self.user = UserFactory()
+        self.client = Client()
```

Comment:
No need to do to this, as its already part of TestCase which is parent of ModuleStoreTestCase.